src/jsdom-patches.js

```diff
@@ -238,3 +238,11 @@ Object.defineProperties( Node.prototype, {
 		},
 	},
 } );
+
+class DOMParser {
+	parseFromString( string ) {
+		return jsdom.html( string );
```

Comment:
Thanks for fixing gb-mobile branch @koke 

For this line, I'm comparing our `parseFromString` inner code: https://github.com/iamcco/jsdom-jscore-rn/blob/a562f3d57c27c13e5bfc8cf82d496e69a3ba2800/lib/jsdom.js#L50-L67 to Gutenberg's `jsdom` `parseFromString` inner code: https://github.com/jsdom/jsdom/blob/d240291edbe7d4180a5152993ced7950c834dc57/lib/jsdom/living/domparsing/DOMParser-impl.js#L12 and they look quite different (hope I'm comparing correct code :-P). For example, Gutenberg `jsdom` disables scripting presumably for security concerns. Perhaps we can add a comment that this simple implementation is being added to accommodate this change: https://github.com/WordPress/gutenberg/pull/18132/, and is not a full-fledged implementation of `DOMParser parseFromString`? 

Also, it seems the parseFromString is only used in a newly added `stripHTML` method, which received some constructive criticism, and advice for an alternative implementation of stripping HTML from the RichText in this comment: https://github.com/WordPress/gutenberg/pull/18132#discussion_r365323192. If we add a comment here, we might be better prepared to remove this code if someone removes the `stripHTML` and `parseFromString` method from Gutenberg code.

Reply:
That's right, I added a comment. I don't see that our implementation of jsdom allows to disable scripting, but it also doesn't seem like a big risk, since we are not rendering into the DOM, and only using this for accessibility labels.